lib/activity/router.js

```diff
@@ -26,7 +27,12 @@ module.exports = function route(callback) {
             include: [GitHubUser],
           });
 
-          if (!await creator.GitHubUser.hasRepoAccess(subscription.githubId)) {
+          const hasRepoAccess = await cache.fetch(
+            subscription.cacheKey('creator-access'),
+            () => creator.GitHubUser.hasRepoAccess(subscription.githubId),
+          );
```

Comment:
What are your thoughts on having a TTL of `10 * 60 * 100` here (10min) given that the hasRepoAccess check is a security feature.

Reply:
👍 That's fine for me. `fetch` takes a ttl as a 3rd arg.